### DISCUSSION TITLE
pipenv shell not found

### DISCUSSION THREAD

**Discussion by Hemazyn** [Atri-Labs/atrilabs-engine#596] [Help: Installation & Start] [answered]
Please I need help with this.
![Screenshot 2022-12-21 at 12 18 26 PM](https://user-images.githubusercontent.com/75551864/208893209-3043dcc5-5d8f-4b40-a232-3ecd4bbf471c.png)

**Comment by darshitac11** [comment, marked as answer]
@Hemazyn - you have not installed `pipenv` which is a prerequisite. Please follow the instructions shared [here](https://docs.atrilabs.com/getting-started/installation/install_prereqs) in the Docs.

**Reply by Hemazyn** [reply]
Good morning ma,
just finish setting up my my environment, I had to download both python and anaconda, update my Xcode. Ma can I get more time to be able to execute the given task, as i was expected to submit it today.

**Comment by Hemazyn** [comment]
![Screenshot 2022-12-21 at 6 09 29 PM](https://user-images.githubusercontent.com/75551864/208963941-96d826d6-1c54-4f63-b7fb-166cfd8a5f50.png)

**Comment by darshitac11** [comment]
@Hemazyn - other prerequisite is Python version and it should be Python >=3.8 & <=3.10.8

There are several problems with Python 3.11 and hence, we do not recommend it.

**Reply by Hemazyn** [reply]
which stable version do you recommend ma.

**Reply by darshitac11** [reply]
As mentioned in my comment above, Python 3.8, 3.9, 3.10 are all good.

**Comment by Hemazyn** [comment]
Good morning ma,
just finish setting up my my environment, I had to download both python and anaconda, update my Xcode. Ma can I get more time to be able to execute the given task, as i was expected to submit it today.
